app/frontend/components/common/balance.js

```diff
@@ -13,7 +13,15 @@ const truncatePrintAda = (amount, maxDigits) => {
   return `${Math.trunc(amountString / 1000)}K`
 }
 
-const Balance = ({balance, reloadWalletInfo}) =>
+const printConversionRates = (amount, conversionRates, maxDigits) =>
+  isNaN(Number(amount))
+    ? ''
+    : maxDigits
+      ? `(${truncatePrintAda(printAda(conversionRates.usd * amount), 12)} $,
```

Comment:
I am pretty sure `truncatePrintAda(printAda(` combo is not what you want for formatting of fiat currencies. maybe create a separate `formatFiat` function that takes in as an agrument number of decimals?